servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -702,6 +702,15 @@ void RenderForwardClustered::_setup_environment(const RenderDataRD *p_render_dat
 
 	scene_state.ubo.pancake_shadows = p_pancake_shadows;
 
+	if (shadows_use_jitter_get() == RS::SHADOW_JITTER_ALWAYS || (shadows_use_jitter_get() == RS::SHADOW_JITTER_AUTO && !p_render_data->taa_jitter.is_equal_approx(Vector2()))) {
```

Comment:
Is there a more efficient way to check whether TAA is enabled?

Also, when TAA is enabled, `taa_jitter` is sometimes `Vector2(0, 0)` (always 4 times in a row, it seems). Maybe this is part of secondary passes though.